### PR TITLE
Project init

### PR DESCRIPTION
**Pull request by PavelTitov16**
Starting project configuration

image

### CODE REVIEW

**Review by turn2river — DISMISSED** [review]

**Review by turn2river — APPROVED** [review]

File: postcss.config.cjs [hunk 1]
```diff
@@ -0,0 +1,7 @@
+module.exports = {
+	plugins: {
+		tailwindcss: {},
+		autoprefixer: {},
+	},
+}
```
Comment (KristiBo) [inline postcss.config.cjs:6]: I think you need to use 2 spaces instead of tabs here

File: src/store/index.ts [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+	initialState,
+	reducers: {
+		increment: (state) => {
+			state.value += 1
+		},
+		decrement: (state) => {
+			state.value -= 1
+		},
+	},
+});
```
Comment (KristiBo) [inline src/store/index.ts:22]: and here replase tabs

File: .prettierrc [hunk 3]
```diff
@@ -0,0 +1,8 @@
+{
+  "semi": false,
```
Comment (KristiBo) [inline .prettierrc]: I prefer to use "semi": true. We can discuss it

**Review by KristiBo — APPROVED** [review]

File: .prettierrc (hunk 3 above)
Comment (PavelTitov16) [reply .prettierrc]: No need - you are right, I'm agree with you

**Pull request merged**